common/net/conn/conn.go

```diff
@@ -0,0 +1,121 @@
+package conn
+
+import (
+	"io"
+	"net"
+)
+
+// MaxBufferSize - size which can store about 1024 frames of AES256
+const MaxBufferSize = 32768
+
+// MinBufferFrameSet - can contains 32 frames of AES256
+const MinBufferFrameSet = 1024
+
+// Conn represents a secured connection.
+// which implements the net.Conn interface.
+type Conn struct {
+	conn   net.Conn
```

Comment:
```suggestion
	net.Conn
```
to inherit all methods:
```go
	// LocalAddr returns the local network address.
	LocalAddr() Addr

	// RemoteAddr returns the remote network address.
	RemoteAddr() Addr
       
        .........
```